### PR TITLE
PLAT-17: Confluent CLI attempts to start or stop control-center in oss flavor

### PR DESCRIPTION
**Pull request by kkonstantine**

### CODE REVIEW

File: src/oss/confluent.sh [hunk 1]
```diff
@@ -913,14 +913,16 @@ start_command() {
     set_or_get_current
     echo "Using CONFLUENT_CURRENT: ${confluent_current}"
     # if specific service is not found, function return 1, so use OR to continue
-    start_or_stop_service "start" "services" "${@}" || start_or_stop_service "start" "enterprise_services" "${@}"
+    start_or_stop_service "start" "services" "${@}" \
+        || is_enterprise && start_or_stop_service "start" "enterprise_services" "${@}"
```
Comment (sansanichfb) [inline src/oss/confluent.sh]: Would it be useful to add debug/info message when starting enterprise components?

Comment (kkonstantine) [reply src/oss/confluent.sh]: We currently don't have separate logs for the cli itself. Anything worth logging is redirected to stdout/stderr, same as other shell commands.

Comment (C0urante) [reply src/oss/confluent.sh]: I think the control flow here isn't quite what we want. This line gets grouped as
```bash
{ start_or_stop_service "start" "services" "${@}" || is_enterprise ; } && start_or_stop_service "start" "enterprise_services"
```
where the last command that starts enterprise services is executed when either command in the `||` statement succeeds.
It looks like the intent is for it to be grouped like
```bash
start_or_stop_service "start" "services" "${@}" || { is_enterprise && start_or_stop_service ; }
```
where the last command that starts enterprise services is executed if and only if the first command (which starts OSS services) fails but the second command (which performs the `is_enterprise` check) succeeds.

Comment (kkonstantine) [reply src/oss/confluent.sh]: That's a good point. Is more easy to confirm in the `service_exists` part. 

I broke the chain of boolean statements.

**Pull request merged**